commit 797aaa00701c833ad383adf22eb8dd18cb312f42 (refs/changes/12/5612/3)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-02-19T16:45:05-08:00 (8 months ago)
    
    TRITON-1242 Create DAPI filter to allow filtering out virtual servers
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index f803b9e..b6908c9 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.7.0",
+  "version": "9.7.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -28,7 +28,7 @@
     "strsplit": "1.0.0",
     "trace-event": "1.3.0",
     "triton-metrics": "0.1.1",
-    "triton-tags": "1.2.2",
+    "triton-tags": "1.3.0",
     "vasync": "2.2.0",
     "verror": "1.10.0",
     "wf-client": "0.3.1"
